### ISSUE TITLE
Add a property to the manifest to designate a component to launch on app startup

### ISSUE THREAD

**Issue by tido64** [microsoft/react-native-test-app#118]
When the app manifest only contains a single component, the test app will try to launch it on startup. There is currently no way to launch an arbitrary component on startup when there are more than one.

**Comment by rohanprasadofficial** [comment]
@tido64 , any approach you have thought for this ?

**Comment by tido64** [comment]
Not really. So far, no one has asked for this so I'm not sure it's actually needed. We also recently added [single-app mode](https://github.com/microsoft/react-native-test-app/wiki/Single-app-Mode) which solves part of the issue.
